### ISSUE TITLE
milc 1.0.4 throws AttributeError when adding subcommand

### ISSUE THREAD

**Issue by Duckle29** [clueboard/milc#7]
This error doesn't happen with 1.0.3

Project using milc: https://github.com/VGIS8/project/blob/master/defector/cli/framediff.py

```
Traceback (most recent call last):
  File ".\bin\defector", line 54, in <module>
    main()
  File ".\bin\defector", line 35, in main
    import defector.cli  # noqa
  File "D:\UNI\8. semester\github\project\defector\cli\__init__.py", line 2, in <module>
    from . import framediff
  File "D:\UNI\8. semester\github\project\defector\cli\framediff.py", line 16, in <module>
    def framediff(cli):
  File "D:\UNI\8. semester\github\project\.venv\lib\site-packages\milc\milc.py", line 214, in argument_function
    self.subcommands[subcommand_name].add_argument(*args, **kwargs)
  File "D:\UNI\8. semester\github\project\.venv\lib\site-packages\milc\configuration.py", line 73, in add_argument
    argument_name = self.cli.get_argument_name(*args, **kwargs)
AttributeError: 'MILC' object has no attribute 'get_argument_name'
```

**Comment by mskymoore** [comment]
Have the same issue when importing milc.subommand.config.

**Comment by skullydazed** [comment]
This should be fixed in 1.0.6.

**Comment by mskymoore** [comment]
@skullydazed  Verified.
